### PR TITLE
Sensu is now notified of deployment

### PR DESCRIPTION
**Pull request by Kyrremann**

### CODE REVIEW

File: cli/cmd/Deploy.go [hunk 1]
```diff
@@ -141,6 +141,8 @@ var deployCmd = &cobra.Command{
 			elapsed := time.Since(start)
 			fmt.Printf("Deploy successful, took %v\n", elapsed)
 		}
+
+		api.NotifyGrafanaAboutDeploy(strings["app"], strings["cluster"], strings["namespace"], strings["version"])
```
Comment (davidsteinsland) [inline cli/cmd/Deploy.go]: Should not be needed as the `deploy` cmd is just sending a http request to naisd deploy

Comment (audunstrand) [reply cli/cmd/Deploy.go]: Agree. This should be server side only

File: api/sensuclient.go [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+		errMsg := fmt.Sprintf("Sensu repsonded with something other than 'ok'. Response was: '%s'", string(buff))
+		return errors.New(errMsg)
+	} else {
+		glog.Info("Notified Sensu about deployment, sent the following message: ", string(message))
+	}
+
+	return nil
+}
+
+func NotifyGrafanaAboutDeploy(application *string, clusterName *string, namespace *string, version *string) {
```
Comment (Kyrremann) [inline api/sensuclient.go]: Should maybe be renamed to `NotifySensuAboutDeploy`, as it's really only communicating with Sensu.

File: api/sensuclient.go [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+	MessageType string   `json:"type"`
+	Handlers    []string `json:"handlers"`
+	Output      string   `json:"output"`
+}
+
+func unixTimeInNano() int64 {
+	return time.Now().UnixNano()
+}
+
+func GenerateDeployMessage(deploymentRequest *NaisDeploymentRequest, clusterName *string) ([]byte, error) {
```
Comment (audunstrand) [inline api/sensuclient.go]: could this be "private"

Comment (Kyrremann) [reply api/sensuclient.go]: Could be, but then I can't test it directly.

Comment (audunstrand) [reply api/sensuclient.go]: ok

**Pull request merged**